spec/features/locales_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe 'locale switching' do
+
+  context 'when arabic is selected' do 
```

Comment:
You say `when arabic is selected` but the code says `visit root_path(:en)`. You shouldn't lie! ;p

* move `visit root_path(:en)` at `describe` level
* Delete this test altogether:

```
     it 'loads page' do
       find("#change-to-arabic").click
       page_ok
     end
```
* Add a `page_ok` at the end of the remaining `it`
* Review the result and make sure it makes sense / adheres to guidelines